### ISSUE TITLE
Use a different tree style for forums posts & discussion comments

### ISSUE THREAD

**Issue by allenwq** [Coursemology/coursemology2#1498]
1. The current way of forum posts display easily reach the nesting limit, which makes the tree meaningless.
2. Cannot do pagination in current case

Both 1 and 2 result in users randomly click the reply button just for replying a post.

**Comment by luoyanjie** [comment]
We can close this, since we have implemented a fix?

**Comment by benleong** [comment]
Li Heng says still got some bug and Wang Qiang is fixing? Once he debugs
and ensures that all is good, then we close?

**Comment by luoyanjie** [comment]
The fix was referenced to another issue. So thought of closing this as it is a duplicate.

**Comment by allenwq** [comment]
Yep, this is a different issue which is not valid after #2444, I will close it.
